tests/unit/test_testutils.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import mock
+
+from awscli.testutils import create_bucket
+from awscli.testutils import BaseCLIDriverTest
+
+ERROR_409 = b'''<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this need to be taken outside the scope of the function? Also, you should probably put junk values for RequestID, HostID, and BucketName. I don't think anything can be done with them, but better safe than sorry (You don't use them anyway, so it's not a big deal).